### PR TITLE
Fix checking value in `study.tell` .

### PR DESCRIPTION
**Pull request by ryota717**
## Motivation

#5175 

## Description of the changes

- 6a082706992a4c5013a2592e27ff545529d3e29e: small fix to treat the single string value as list of single string.
- e52a96a1d6ea7f8dd159351769b7d7fa83b0fb01: fix the problem mentioned in the issue.

### CODE REVIEW

File: optuna/study/_tell.py [hunk 1]
```diff
@@ -118,7 +117,7 @@ def _tell_with_warning(
     values: Optional[Sequence[float]]
     if value_or_values is None:
         values = None
-    elif isinstance(value_or_values, Sequence):
+    elif not isinstance(value_or_values, str) and isinstance(value_or_values, Sequence):
```
Comment (contramundum53) [inline optuna/study/_tell.py:120]: Is this change necessary?

Comment (contramundum53) [reply optuna/study/_tell.py:120]: I see, you are trying to make the error readable 👍

**Review by knshnb — APPROVED** [review]
Thanks for the PR! LGTM.

**Review by contramundum53 — APPROVED** [review]
Sorry for late reply, LGTM!
